LinkedListPalindrome/LinkedListPalindrome.js

```diff
@@ -0,0 +1,43 @@
+class ListNode {
+    constructor(val, next = null) {
+      this.val = val;
+      this.next = next;
+    }
```

Comment:
Although you can do this, setting variables to `null` is a very Java thing. If you don't pass an argument to a function call, JS will use `undefined` by default.
```suggestion
    constructor(val, next) {
      this.val = val;
      this.next = next;
    }
```